install/0000_30_machine-api-operator_13_machinehealthcheck.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  name: machine-api-termination-handler
+  namespace: openshift-machine-api
+  labels:
+    api: clsuterapi
+    k8s-app: termination-handler
+spec:
+  selector:
+    matchLabels:
+      machine.openshift.io/interruptible-instance: ""
+  maxUnhealthy: 100%
```

Comment:
does this mean that it would tolerate 100% of the worker nodes being unhealthy, or is this just for spot instances?

Reply:
This is 100% so that all spot instances that are marked with the terminating conditions (ie ones that will go away within 2 minutes or 30 seconds depending on provider), are deleted and drained immediately, there's no point trying to not interrupt the state of the cluster anymore, as it's going to be interrupted anyway